### ISSUE TITLE
Jenkins isn't sending any notifications

### ISSUE THREAD

**Issue by alexbilbie** [jlewallen/jenkins-hipchat-plugin#65]
Hello,

I've installed the plugin and configured it but I am not receiving any notifications in Hipchat.

Can you please clarify which API key I should be using - one generated from Group Admin -> API, or one from a users's Account Settings -> API access page?

Is there a log file somewhere that I can tail to try and diagnose the issue?

Thank you

**Comment by pwyliu** [comment]
Yeah, I couldn't get it to work either. I think there is something wrong with version 0.1.5, but I didn't know how to debug further.

I got mine working now. To fix it, I removed the plugin and settings file, then manually installed 0.1.4 from [here](https://updates.jenkins-ci.org/download/plugins/hipchat/). Then I removed and readded the post build action from all my projects, just to be sure. I'm using a V1 API key.

Edit: actually I should mention I had this problem on two separate jenkins servers where I was installing the hipchat plugin for the first time. 0.1.5 from the plugin manager didn't work, but manually installing 0.1.4 does.
